cowamm/kpi/tvl_by_chain_4096107.sql

```diff
@@ -0,0 +1,85 @@
+-- This query computes the total TVL evolution per day for all CoW AMM pools for a given chain
+-- Parameters
+--  {{blockchain}} - the target chain
+
+with pools as (
+    select
+        created_at,
+        token_1_address,
+        token_2_address,
+        address
+    from query_3959044
+),
+
+date_range as (
```

Comment:
Should we distinguish between mainnet/gnosis and Arbitrum, which launched later?

Reply:
I think it's fine to start the TVL analysis at the earliest possible day (it will just be 0 for chains that haven't launched yet)